Makefile

```
@@ -52,4 +52,16 @@ compile_translations:
 translate:
 	pipenv run pybabel extract -F babel.cfg -o rh_ui/translations/messages.pot . 		# update the .pot files basing on templates
 	pipenv run pybabel update -i rh_ui/translations/messages.pot -d rh_ui/translations	# update .po files basing on .pot
-	pipenv run pybabel compile -d rh_ui/translations
\ No newline at end of file
+	pipenv run pybabel compile -d rh_ui/translations
+
+up:
+	docker compose up -d
+	bash ./tests/integration/wait_for_dependencies.sh
+
+down:
+	docker compose down
+
+integration_test: lint up
```

```suggestion
integration_test: up
```
Don't need to run linting again